build.gradle

```diff
@@ -187,10 +187,10 @@ rat {
   failOnError = true
   excludes = [
     // Exclude files generated by the Gradle build process
-    "**/.gradle/**",
-    "**/.gogradle/**",
-    "**/build/**",
-    "**/vendor/**",
+    "**/.gradle/**/*",
+    "**/.gogradle/**/*",
+    "**/build/**/*",
+    "**/vendor/**/*",
     "**/.gradletasknamecache",
 
     // .gitignore: Ignore files generated by the Maven build process
```

Comment:
remove the duplicate `**/build/**/*` directory below like you did in the .gitignore

Also, `sdks/python/**/*.egg` is missing from the list below